docs/resources/job.md

```diff
@@ -3,63 +3,45 @@
 page_title: "qovery_job Resource - terraform-provider-qovery"
 subcategory: ""
 description: |-
-  Provides a Qovery job resource. This can be used to create and manage Qovery job registry.
+  Provides a Qovery job resource. This can be used to create and manage Qovery job registry. Cron job example https://github.com/Qovery/terraform-examples/tree/main/examples/deploy-a-cron-job Lifecycle job example https://github.com/Qovery/terraform-examples/tree/main/examples/deploy-a-lifecycle-job
 ---
 
 # qovery_job (Resource)
 
-Provides a Qovery job resource. This can be used to create and manage Qovery job registry.
+Provides a Qovery job resource. This can be used to create and manage Qovery job registry. [Cron job example](https://github.com/Qovery/terraform-examples/tree/main/examples/deploy-a-cron-job) [Lifecycle job example](https://github.com/Qovery/terraform-examples/tree/main/examples/deploy-a-lifecycle-job)
 
 ## Example Usage
 
 ```terraform
 resource "qovery_job" "my_job" {
-  # Required
-  environment_id = qovery_environment.my_environment.id
-  name           = "test-job"
-
-  # Optional
-  auto_preview         = "true"
-  cpu                  = 500
-  memory               = 512
-  max_duration_seconds = 23
-  max_nb_restart       = 1
-  port                 = 5432
-  environment_variables = [
-    {
-      key   = "MY_TERRAFORM_CONTAINER_VARIABLE"
-      value = "MY_TERRAFORM_CONTAINER_VARIABLE_VALUE"
-    }
-  ]
-  secrets = [
-    {
-      key   = "MY_TERRAFORM_CONTAINER_SECRET"
-      value = "MY_TERRAFORM_CONTAINER_SECRET_VALUE"
-    }
-  ]
+  environment_id = "<my-environment-id>"
+  name           = "<my-job-name>"
+
   schedule = {
     on_start  = {}
     on_stop   = {}
     on_delete = {}
     cronjob = {
-      schedule = "*/2 * * * *"
+      schedule = "* * * * *"
       command = {
-        entrypoint = ""
-        arguments  = ["echo", "'DONE'"]
+        entrypoint = "<my-job-entrypoint>"
+        arguments  = ["<my-job-argument-1>", "<my-job-argument-2>"]
       }
     }
   }
+
   source = {
     image = {
-      registry_id = qovery_container_registry.my_container_registry.id
-      name        = "debian"
-      tag         = "stable"
+      registry_id = "<my-Qovery-container-registry-id>"
+      name        = "<my-image-name>"
+      tag         = "<my-image-tag>"
+    }
+    # or
+    docker = {
+      git_repository  = "<my-git-repository-url>"
+      dockerfile_path = "<my-dockerfile-path-in-the-repo>"
     }
   }
-
-  depends_on = [
```

Comment:
why remove this?

Reply:
Because it's not relevant at all in a standalone example. Depends_on means you have declared another resource in your tf file which isn't the case here.

Reply:
makes sense